django/views/csrf.py

```diff
@@ -97,12 +97,11 @@
 """
 
 
-def csrf_failure(request, reason=""):
+def csrf_failure(request, reason="", template_name="403_csrf.html"):
```

Comment:
Minor issue, but I would prefer using `None` as the argument default, and then using `get_template(template_name or "403_csrf.html")` when loading the template, and then using `if template_name is None` in the `else` clause. I think this is what @charettes initially suggested? The advantage is just that we avoid hardcoding the default template name in two different places.